Updated Catalog UI configurations

jlcsmith opened the pull request:
#### What does this PR do?
- removed blueprint default configurations
- added catalog UI config file with default configurations
- updated hidden attribute config to include workspace and query attributes

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@rzwiefel @kcwire @andrewkfiedler 

#### How should this be tested? (List steps with links to updated documentation)

Build and verify catalog UI app installs
Verify default configuration exists and is correct after install
Verify hidden attribute configuration is correct

#### Any background context you want to provide?
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Screenshots (if appropriate)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

jvettraino commented:
Thanks @shaundmorris, my bad.  Also should I create one for the bug fix branch, or will you cherry-pick?

shaundmorris commented:
@jvettraino  I will cherry-pick it for you. no problem there

Commit:
Updated Catalog UI configurations
- removed blueprint default configurations
- added catalog UI config file with default configurations
- updated hidden attribute config to include workspace and query attributes

rzwiefel commented:
Built and ran successfully
